misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathAssertionsSpec.kt

```diff
@@ -134,36 +131,33 @@ abstract class PathAssertionsSpec(
         val endsWithFun = endsWith.lambda
         val endsNotWithFun = endsNotWith.lambda
 
-        context("ends with") {
-            it("${endsWith.name} does not throw") {
-                expect(Paths.get("/not/existed/for/test"))
+        context("path '/some/path/for/test'") {
```

Comment:
no need for the context as it is the same as above

Reply:
okay. 👍